doc/content/guides/getting-started/configuration.md

```diff
@@ -4,29 +4,158 @@ description: ""
 weight: 3
 ---
 
-The Things Stack can be started for development without passing any configuration. However, there are a lot of things you can configure. In this guide we'll set some environment variables in `docker-compose.yml`. These environment variables will configure The Things Stack as a development server on `localhost`. For setting up up a public server or for requesting certificates from an ACME provider such as Let's Encrypt, take a closer look at the comments in `docker-compose.yml`.
+The Things Stack can be started for development without passing any configuration. However, there are a number of things you need to configure for a production deployment. In this guide we'll set environment variables in the `docker-compose.yml` file to configure The Things Stack as a private deployment on `your.thethingsstack.io`.
```

Comment:
The `docker-compose.yml` in the repository can be used as a base, then it's just a matter of (un)commenting and editing lines. I still think it's good to quickly go through everything in the file.

Reply:
I think it would be nice to eventually have a wizard (`ttn-lw-cli quickstart`) that would try to detect the environment (Docker, ...), ask some questions, write everything to a config file, initialize the database, set up the user and register the OAuth clients. That would be something for later though.

Reply:
> The `docker-compose.yml` in the repository can be used as a base, then it's just a matter of (un)commenting and editing lines. I still think it's good to quickly go through everything in the file.

Yes that's what I thought. What's missing then is the link to that file.

Reply:
We can't really link to that file, since it may be release-specific. The documentation is bundled with the release archive, so if the reader downloaded an archive, they already have it. If they're looking at the online documentation, then we need to link to a specific version, which would require a shortcode to fill that version.

Of course we can pull the yml into the `doc` folder on build, or add a shortcode to fill a (perma)link to a specific version of the docker-compose file, but I'm not sure if we want to do all that in this PR.

Reply:
I understand the technical reason why we cannot simply put a link here now, but this is a Getting Started and is supposed to make things easy. We'll be pointing directly from The Things Stack landing page to this Getting Started, so nobody downloaded a release yet. 

My concern is mostly with what's in **Installation** and where we build upon here, when configuring;

>Most releases contain an example `docker-compose.yml` file. You can also find this file in the Github repository of The Things Stack. In this guide we’ll use that example `docker-compose.yml` for our deployment.

There's not even a link to the GitHub repository here. Note that, for a lot of people, GitHub is quite intimidating, so figuring out that there is a `TheThingsNetwork` account, that there's a `lorawan-stack` repository in there, and then downloading the raw file of `docker-compose.yml`, is at best good natural selection for skilled people installing the stack, and at worst a way for people to get lost even before they started.

I would, in fact, suggest copying in `docker-compose.yml` with the exact purpose of a template for this Getting Started. We may even refer to documentation steps, which we cannot really do in the `docker-compose.yml` we keep in the root.

Reply:
Added direct link to the file in 3d5c0f42987a6c532f9c2a8f4959694cb6abdf03